README.md

```diff
@@ -115,26 +161,21 @@ Thanks goes to these wonderful people
     <td align="center"><a href="https://github.com/tuledu/"><img src="https://avatars.githubusercontent.com/u/111717882?v=4" width="100px;" alt="Tung Le" style="border-radius: 10%;"/><br /><sub><b>Tung Le</b></sub></a><br /></td>
   </tr>
 </table>
-## Documents
-- [Config redux persist](docs/config-redux-persist.md)
+
 ## Libraries used:
-```text
-    - redux-persist
-    - @react-native-async-storage/async-storage
-    - @react-navigation/bottom-tabs
-    - @react-navigation/native
-    - @react-navigation/native-stack
-    - react-native-code-push
-    - react-native-fast-image
-    - react-native-gesture-handler
-    - react-native-progress
-    - react-native-reanimated
-    - react-native-safe-area-context
-    - react-native-screens
-    - react-redux
-    - redux-saga
-    - i18n-js
-```
+- [Redux Persist](https://github.com/rt2zz/redux-persist) 
+- [React Native Async Storage](https://github.com/react-native-async-storage/async-storage)
+- [React Navigation](https://github.com/react-navigation/react-navigation?tab=readme-ov-file)
+- [React Native Code Push](https://github.com/microsoft/react-native-code-push)
+- [React Native Fast Image](https://github.com/DylanVann/react-native-fast-image)
+- [React Native Gesture handle](https://github.com/software-mansion/react-native-gesture-handler)
+- [React Native Progress](https://github.com/oblador/react-native-progress)
+- [React Native Reanimated](https://github.com/software-mansion/react-native-reanimated)
+- [React Native Safe Area Context](https://github.com/th3rdwave/react-native-safe-area-context)
+- [React Native Screens](https://github.com/software-mansion/react-native-screens)
+- [React Redux](https://github.com/reduxjs/react-redux)
+- [React Saga](https://github.com/redux-saga/redux-saga)
+- [I18n JS](https://github.com/fnando/i18n-js)
```

Comment:
I don't think we need this section. I recall that we removed it in the project javascript

Reply:
oke updated